### PR TITLE
Nodegroup taints support

### PR DESCRIPTION
**Pull request by dstdfx**
For #49

### CODE REVIEW

File: pkg/v1/cluster/testing/fixtures.go [hunk 1]
```diff
@@ -854,7 +854,14 @@ const testCreateClusterOptsRaw = `
...
                   "test-label-key": "test-label-value"
-                }
+                },
+                "taints": [
+                  {
+                    "key": "test-key-0",
+                    "value": "test-value-0",
+                    "effect": "NoSchedule"
+                  }
+		        ]
```
Comment (ozerovandrei) [inline pkg/v1/cluster/testing/fixtures.go:864]: Bad indent.

**Review by ozerovandrei — CHANGES_REQUESTED** [review]

File: pkg/v1/cluster/doc.go [hunk 2]
```diff
@@ -40,6 +40,14 @@ Example of creating a new cluster
           "label-key1": "label-value1",
           "label-key2": "label-value2",
         },
+        Taints: []nodegroup.Taint{
+            {
```
Comment (kolsean) [inline pkg/v1/cluster/doc.go:44]: Let's use two spaces for indentation here in line with other examples.

File: pkg/v1/cluster/doc.go [hunk 3]
```diff
@@ -40,6 +40,14 @@ Example of creating a new cluster
...
+                Key:    "test-key-0",
+                Value:  "test-value-0",
+                Effect: nodegroup.NoScheduleEffect,
+            },
+          },
+        },
```
Comment (kolsean) [inline pkg/v1/cluster/doc.go:50]: extra `},` can be removed

**Review by kolsean — CHANGES_REQUESTED** [review]
fix doc

File: pkg/v1/nodegroup/testing/fixtures.go [hunk 4]
```diff
@@ -32,7 +32,14 @@ const testGetNodegroupResponseRaw = `
...
            "test-label-key": "test-label-value"
-        }
+        },
+        "taints": [
+            {
+                "key": "test-key-0",
+                "value": "test-value-0",
+                "effect": "NoSchedule"
+            }
+       ]
```
Comment (ozerovandrei) [inline pkg/v1/nodegroup/testing/fixtures.go:42]: Bad indent.

**Review by ozerovandrei — CHANGES_REQUESTED** [review]

File: pkg/v1/cluster/doc.go [hunk 5]
```diff
@@ -40,6 +40,13 @@ Example of creating a new cluster
           "label-key1": "label-value1",
           "label-key2": "label-value2",
         },
+        Taints: []nodegroup.Taint{
+          {
+                Key:    "test-key-0",
+                Value:  "test-value-0",
+                Effect: nodegroup.NoScheduleEffect,
```
Comment (kolsean) [inline pkg/v1/cluster/doc.go:47]: It would be great to use two spaces here too.

**Review by kolsean — CHANGES_REQUESTED** [review]

**Review by ozerovandrei — APPROVED** [review]
👍

**Review by kolsean — APPROVED** [review]
LGTM!

**Pull request merged**